main/speech/tts_wrapper.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# This script is the wrapper script for TTS.
+# This script is required to process multiple queries simultanously
+
+SCRIPT_PATH=$(realpath $0)
+DIR_PATH=$(dirname $SCRIPT_PATH)
+
+python3 $DIR_PATH/TTS.py $1 $2
```

Comment:
Are you passing which services for TTS through the command line arguments here.
Like google or flite?

Reply:
Yes @prateekiiest